src/components/atoms/deploymentStatusBadge.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import { DeploymentStatusBadgeProps } from "@interfaces/components";
+import { DeploymentStateVariant } from "@src/enums";
+import { cn } from "@utilities";
+
+export const DeploymentStatusBadge = ({ className, deploymentStatus }: DeploymentStatusBadgeProps) => {
+	const baseClass =
+		"w-20 flex align-center justify-center font-fira-code rounded-md p-1 px-1.5 font-semibold tracking-widest capitalize";
+
+	const badgeClass = cn(
+		baseClass,
+		{
+			"border border-dashed border-gray-500 text-gray-500 text-10":
+				deploymentStatus === DeploymentStateVariant.draining,
+			"bg-black border border-white text-white text-10": deploymentStatus === DeploymentStateVariant.active,
+			"bg-gray-1050 text-gray-500 text-10": deploymentStatus === DeploymentStateVariant.inactive,
```

Comment:
We need to keep `text-10` in **active**, because `text-10` is a custom tailwind size, and when we do `cn(text-10 text-white)` the result is only `text-white` without the size of it.

Reply:
You can try... It won't work because of the `cn` function

Reply:
yes, you're right

Reply:
<img src="https://media0.giphy.com/media/d3mlE7uhX8KFgEmY/giphy.gif"/>